source/_posts/Github-workflow.md

```diff
@@ -0,0 +1,108 @@
+---
+title: Github Workflow
+desc: A mostly reasonable approach Github and Git
+date: 2018-07-15 16:14:30
+categories: code-standards
+permalink: github-workflow
+---
+
+## Workflow
+
+Currently, there are two type of projects, some are in production stage and other in the early stage.
+
+### Projects in Production
+
+For projects that are in production, that means projects that have a scheduled release live cycle, we could work with Git Flow:
+
+[Learn Git flow](https://www.atlassian.com/git/tutorials/comparing-workflows/gitflow-workflow) *by Atlassian*
+
+
+### Projects in Early stage
+
+For projects that is in the early stage, that means not already in production, we could work with Feature Branch Workflow:
+
+[learn Feature Branch Workflow](https://www.atlassian.com/git/tutorials/comparing-workflows/feature-branch-workflow) *by Atlassian*
+
+## Good commit messages
```

Comment:
WRT to commit conventions... we have so far tried to follow/adopt the popular Angular Commit Conventions: https://gist.github.com/stephenparish/9941e89d80e2bc58a153

It might be a good idea to consider a blend between the above mentioned conventions and the ones outlined in this PR. Thoughts?

Reply:
I liked this convention 👍 I will keep the discussion on #13